File: Docility-SPA/src/app/_models/communication.ts
import { User } from './user';
import { Workgroup } from './workgroup';

export interface Communication {
    id: number;
    message: string;
    user?: User;
    userId: number;
    bugId: number;
    isPrivate: boolean;
    workgroup?: Workgroup;
    timeStamp: Date;
}

File: Docility.API/Models/Communication.cs
using System;

namespace Docility.API.Models
{
    public class Communication
    {
        public int Id { get; set; }
        public string Message { get; set; }
        public User User { get; set; }
        public Bug Bug { get; set; }
        public int BugId { get; set; }
        public bool IsPrivate { get; set; }
        public Workgroup Workgroup { get; set; }
        public DateTime TimeStamp { get; set; } = DateTime.Now;
    }
}